EXPLAINER.md

```diff
@@ -234,7 +233,7 @@ The reader and writer objects could have been merged into one single object, but
 as we allow multiple scans with filters to be active at the same time (maybe in
 multiple places/view of the app/site) then it makes more sense to be able to use
 separate objects.
- 
+
 ## Considered alternatives
```

Comment:
Please remove `### Separate objects for reader/writer` section above.

Reply:
Yes.

Reply:
Wait, that's only a title for a problem (explaining the API choice). So what should be the new name?

Reply:
Updated